apps/pie-microsite/src/_11ty/shortcodes/list.js

```diff
@@ -0,0 +1,26 @@
+const markdownFilter = require('../filters/markdown');
+const listTypes = require('../../_data/listTypes');
```

Comment:
I may be wrong _but_ I think you can access any global 11ty data through `this` within the shortcode function (but not nested functions called inside it). If that works, you can remove the import and it would be something like `this.data.listTypes` (actual syntax may vary - I think the docs mention this)